topics/sequence-analysis/tutorials/quality-contamination-control/tutorial.md

```diff
@@ -0,0 +1,564 @@
+
+   1. Percentage of fragments covered by the clade rooted at this taxon
+   2. Number of fragments covered by the clade rooted at this taxon
+   3. Number of fragments assigned directly to this taxon
+   4. A rank code, indicating
+      - (U)nclassified
+      - (R)oot
+      - (D)omain
+      - (K)ingdom
+      - (P)hylum
+      - (C)lass
+      - (O)rder
+      - (F)amily
+      - (G)enus, or
+      - (S)pecies
+   5. NCBI taxonomic ID number
+   6. Indented scientific name
+
+   ```
+   Column 1	Column 2	Column 3	Column 4	Column 5	Column 6
+   100.00 	450408 	0 	R 	1 	root
+   99.14 	446538 	0 	R1 	131567 	cellular organisms
+   99.14 	446524 	0 	D 	2 	Bacteria
+   99.14 	446524 	0 	D1 	1783272 	Terrabacteria group
+   99.13 	446491 	0 	P 	1239 	Firmicutes
+   99.13 	446491 	0 	C 	91061 	Bacilli
+   99.06 	446152 	0 	O 	1385 	Bacillales
+   99.06 	446152 	0 	F 	90964 	Staphylococcaceae
+   99.04 	446101 	0 	G 	1279 	Staphylococcus
+   95.62 	430661 	430661 	S 	1280 	Staphylococcus aureus 
+   ```
+
+   > <question-title></question-title>
+   >
+   > 1. How many taxa have been found?
+   > 2. What is the family found?
+   >
+   > > <solution-title></solution-title>
+   > >
+   > > 1. 119, as the number of lines
+   > > 2. Staphylococcus, with 99.04%!
+   > {: .solution}
+   >
+   {: .question}
+
+- **Report**: tabular files with one line per taxon and 7 columns or fields
+
+   1. Taxon name
+   2. Taxonomy ID
+   3. Level ID (S=Species, G=Genus, O=Order, F=Family, P=Phylum, K=Kingdom)
+   4. Kraken assigned reads
+   5. Added reads with abundance re-estimation
+   6. Total reads after abundance re-estimation
+   7. Fraction of total reads
+
+   ```
+   Column 1	Column 2	Column 3	Column 4	Column 5	Column 6	Column 7
+   Staphylococcus aureus 	1280 	S 	182995 	247666 	430661 	0.95616
+   Staphylococcus roterodami 	2699836 	S 	698 	48 	746 	0.00166
+   Staphylococcus epidermidis 	1282 	S 	587 	13 	600 	0.00133
+   Staphylococcus lugdunensis 	28035 	S 	511 	3 	514 	0.00114
+   Staphylococcus schweitzeri 	1654388 	S 	409 	26 	435 	0.00097
+   Staphylococcus simiae 	308354 	S 	398 	2 	400 	0.00089 
+   ```
+
+> <question-title></question-title>
+>
+> 1. How many species have been found?
+> 2. Which the species has been the most identified? And in which proportion?
+> 3. What are the other species?
+>
+> > <solution-title></solution-title>
+> > 1. 51 (52 lines including 1 line with header)
+> > 2. *Staphylococcus aureus* with 95.6% of the reads.
+> > 3. Most of the other species are from *Staphylococcus* genus, so same as *Staphylococcus aureus*. The other species in really low proportion.
+> {: .solution}
+{: .question}
+
+As expected *Staphylococcus aureus* represents most of the reads in the data.
+
+# Contamination identification
+
+To explore **Kraken** report and specially to detect more reliably minority organisms or contamination, we will use **Recentrifuge** ({% cite marti2019recentrifuge %}).
```

Comment:
It might be good what is the added value of Recentrifuge (e.g. compared to Bracken) and how it works

Reply:
And maybe it makes more sense to have Recentrifuge before Bracken

Reply:
I prefer to leave Recentrifuge in the same place, it makes more sense (for me) :cry: